community/modules/network/rdma-vpc/outputs.tf

```diff
@@ -43,3 +43,17 @@ output "subnetwork_interfaces" {
   value       = local.output_subnets
   depends_on  = [module.vpc]
 }
+
+# The output subnetwork_interfaces is compatible with vm-instance module but not with gke-node-pool
+# See https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/99493df21cecf6a092c45298bf7a45e0343cf622/modules/compute/vm-instance/variables.tf#L220
+# So, we need a separate output that makes the network and subnetwork names available
+output "subnetwork_interfaces_gke" {
+  description = "Full list of subnetwork objects belonging to the new VPC network (compatible with gke-node-pool)"
+  value       = local.output_subnets_gke
+  depends_on  = [module.vpc]
+}
+
+output "subnetwork_name_prefix" {
```

Comment:
We need this in the `gke-cluster` module to prepare the pod network manifests. 

Input to the kubectl-apply module needs all the map values be static. This is an input to the rdma-vpc module. We need this value in the gke-cluster module as well.

Otherwise we run into

```sh
Error: Invalid count argument

  on modules/embedded/modules/management/kubectl-apply/kubectl/main.tf line 50, in data "kubectl_path_documents" "yamls":
  50:   count   = local.directory != null ? 1 : 0

The "count" value depends on resource attributes that cannot be determined
until apply, so Terraform cannot predict how many instances will be created.
To work around this, use the -target argument to first apply only the
resources that the count depends on.

```